packages/telemetry/lib/telemetry.js

```diff
@@ -117,6 +117,12 @@ async function setupTelemetry (app, opts) {
   app.decorateRequest('span')
 
   const startSpan = async (request) => {
+    const SKIP_OPERATIONS = ['GET/documentation/json']
```

Comment:
Added as configurable. Since we don't have an option to set  telemetry when creating service (yet) we cannot add the option to skip `GET/documentation/json` in default service configuration.